src/navigation-plan.js

```diff
@@ -51,10 +51,10 @@ export function _buildNavigationPlan(navigationContext: NavigationContext, force
       if (prevViewPortInstruction.moduleId !== nextViewPortConfig.moduleId) {
         viewPortPlan.strategy = activationStrategy.replace;
       } else if ('determineActivationStrategy' in prevViewPortInstruction.component.viewModel) {
-         //TODO: should we tell them if the parent had a lifecycle min change?
```

Comment:
@EisenbergEffect Do you know what this means? I removed it because it doesn't seem useful to keep in the code, but if there's some action needed to resolve it I'm happy to check it out.